apiserver/deploy/base/secure/kustomization.yaml

```diff
@@ -6,9 +6,10 @@ namespace: ray-system
 resources:
 - apiserver.yaml
 
-commonLabels:
-  app.kubernetes.io/name: kuberay
-  app.kubernetes.io/component: kuberay-apiserver
+labels:
+  - pairs:
+      app.kubernetes.io/name: kuberay
+      app.kubernetes.io/part-of: kuberay-apiserver
```

Comment:
ditto, all `labels` fields have the same problem as the above.

Reply:
this one also fixed, thanks